### PR TITLE
Add pynput-robocorp-fork and rpaframework

### PR DESCRIPTION
**Pull request by xylix**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by wolfv** [comment]
I have checked the failures, and they are both just in the pip check of the test section:

- OS X has a problem with lowercase `Cocoa`, doesn't seem like an actual issue
- On win there is a confusion with pywin32 and pypiwin32 (seems to be fine to use pywin32).

I think you can ignore these issues and comment out `pip check` for win and osx, or remove it entirely.

**Comment by xylix** [comment]
Testing only running pip-check on linux now

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -65,6 +65,7 @@ about:
   # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
   license_file: LICENSE.txt
   summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+
```
Comment (wolfv) [inline recipes/example/meta.yaml:68]: maybe don't change the example?

File: recipes/rpaframework/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    - openpyxl >=3.0.3,<4.0.0
+    - pdfminer.six ==20201018
+    - pillow >=8.0.1,<9.0.0
+    - psutil >=5.7.0,<6.0.0  # [win]
+    - pynput-robocorp-fork >=2.0.0,<3.0.0
+    - pypdf2 >=1.26.0,<2.0.0
+    - pyperclip >=1.8.0,<2.0.0
+    - python
+    - python-xlib >=0.17  # [linux]
+    - pywin32 >=227,<228  # [py<37 and win or py>37 and py<38 and win or py>38 and win]
```
Comment (wolfv) [inline recipes/rpaframework/meta.yaml:45]: can the condition be `[py>=37 and win]`?

**Comment by wolfv** [comment]
I am confused about the latest linter message as I cannot see any reference to xeus-sql. Maybe because the example was touched?

Looks good to me otherwise!

File: recipes/rpaframework/meta.yaml (hunk 2 above)
Comment (xylix) [reply recipes/rpaframework/meta.yaml:45]: That would drop the dependency for python versions <3.7, right? But that is actually not a problem for us so we can try that one.

Comment (xylix) [reply recipes/rpaframework/meta.yaml:45]: Maybe it could even be `[py<=40 and win]`?

**Comment by xylix** [comment]
Testing if undoing the modification of example will avoid weird linter error (and also it is best practice to not modify the example. I'm unsure when it was actually modified, it was probably accidental).

**Pull request merged**